src/Symfony/Bridge/Twig/Attribute/Template.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Attribute;
+
+#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_FUNCTION)]
+class Template
+{
+    public function __construct(
+        public string $template,
+        public ?array $vars = null,
+        public bool $stream = false,
```

Comment:
What about a `statusCode` property?

Reply:
This has not been requested on extra-bundle along the years.
I'd prefer keeping this PR minimal and improve later on, *if needed*.

Reply:
@HeahDude if needed, this *should* be an other attribute `#[HttpStatus(410)]`, not tied to Twig.